WeakAuras/Types.lua

```diff
@@ -6,6 +6,11 @@ local LSM = LibStub("LibSharedMedia-3.0");
 local wipe, tinsert = wipe, tinsert
 local GetNumShapeshiftForms, GetShapeshiftFormInfo = GetNumShapeshiftForms, GetShapeshiftFormInfo
 local GetNumSpecializationsForClassID, GetSpecializationInfoForClassID = GetNumSpecializationsForClassID, GetSpecializationInfoForClassID
+local WrapTextInColorCode, GetClassColor = WrapTextInColorCode, GetClassColor -- for Classic
+
+-- shouldn't have to redeclare this here because it's already in Init.lua but it wasnt working ...
```

Comment:
Have you investigated this since doing the workaround?

Reply:
I removed the workaround and it's fine now, maybe turning WeakAuras.IsClassic into a function was the solution, i don't know what it change